pymedphys/tests/e2e/cypress/integration/streamlit/pseudonymisation.spec.js

```diff
@@ -0,0 +1,256 @@
+/**
+ * @license
```

Comment:
We should open an issue for cleaning out the cruft and updating the copyright.
While this started from a file from streamlit, it's got precious little left of that.
I will leave the license where it is, but above it indicate that it started from streamlit but was heavily modified, and include both Simon and Stuart (you and me if 3rd person sounds weird)